bundles/org.openhab.binding.knx/README.md

```diff
@@ -60,7 +60,8 @@ The *serial* bridge accepts the following configuration parameters:
 
 *basic* Things are wrappers around arbitrary group addresses on the KNX bus.
 They have no specific function in the KNX binding, except that if the *address* is defined, the binding will actively poll the Individual Address on the KNX bus to detect that the KNX actuator is reachable.
-Under normal real-world circumstances, either all devices on a bus are reachable, or the entire bus is down.
+Under normal real-world circumstances, either all devices on a bus are reachable, or the entire bus is down. If line couplers are installed,
+physical device adressing might be filtered; in this case please do not specify the addresses for devices on this line.
```

Comment:
The convention is to go to a new line when starting a new sentence and not inside a sentence. So it would be better to go to a new line just before "If line couplers" and not just before "physical device adressing".
And please fix typo: addressing instead of adressing

Reply:
Ok, did not remember this convention. Done.